### DISCUSSION TITLE
Multiple SMTP servers in postal.yml per organization

### DISCUSSION THREAD

**Discussion by alex-pommier** [postalserver/postal#2107] [Help with using Postal]
The postal webGUI allows for multiple Organizations and SMTP servers to be added however despite configuring a whole different set of IP's for ORG1 and ORG2 emails are always sent from ORG1's IPs.

Does postal have the ability to deliver emails from smtp1.mydomain.com for ORG1 and smtp2.myseconddomain.com for ORG2. If so what are the necessary configurations that need to be made for that to work as described?

In the postal.yml file there is a section for SMTP_Servers can I add a secondary server there? Currently I have 8 IP's on 1 NIC on the machine but I could split it into 2 nic's with 4 IP's each and have them bound separately is this possible?

**Comment by willpower232** [comment]
have you had a look at IP Pools? https://docs.postalserver.io/features/ip-pools

**Reply by alex-pommier** [reply]
Yes, the IP Pools are setup and IP's are setup correct on the NIC of the machine I can telnet to postal via all 4 IP's on port 25. What I was trying to accomplish is have 2 separate  SMTP addresses within postal so that the corresponding Postal SMTP's have their own hostnames when they send out emails. I'm not sure this is possible however because even though you can configure IP pools the emails sent out are always with the hostname of the server regardless of which domain/postal servers are configured in the webgui. I initially though if I added a secondary NIC with it's own set of IP's that I could have 2 smtp postal servers sending on different hostnames. I ended up spinning up a second VM for my 2nd domain.

IE:
If I'm sending emails through postal to gmail then gmail sees:
Connection request from mail.company1.com

I would like gmail to see connection requests from 
mail.company1.com
When using Company1's ORG/server in postal
&
mail.company2.com
When using Company2's ORG/server in postal

NOT
sender is account@company2.com
but connection is coming from mail.company1.com

Hopefully that makes sense.

**Reply by alex-pommier** [reply]
What threw me off was the ability to add multiple organizations/servers within the postal interface it gives the impression you can have multiple unique sending hostnames using the IP pools but I think the intention is more to able to control client's usage of postal and sending via your postal hostname but rewriting the sender from so it arrives with any email you'd like but the underlying server connections all happen from only 1 domain.

**Reply by willpower232** [reply]
Your explanation makes sense but I'm confused because the whole point of IP Pools is to specifically send from certain IP addresses from certain Organisations.

Unfortunately I don't use IP Pools so I only have a vague understanding of them to go on.
